### PR TITLE
Fix artifacts in gosec target

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
Moves the junit file to correct directory. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
"NONE"
```

### CODE REVIEW

File: hack/gosec.sh [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 source hack/common.sh
-export ARTIFACTS=${ARTIFACTS:-$KUBEVIRT_DIR/_out/artifacts}
 
 mkdir -p $ARTIFACTS
 
 echo "Run go sec in pkg"
 cd $KUBEVIRT_DIR/pkg
 
-gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
+gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
```
Comment (enp0s3) [inline hack/gosec.sh:10]: @xpivarc Just a small one - can you please add a newline at the end?

**Review by enp0s3 — COMMENTED** [review]
Looks great! Just a small nit below

File: hack/gosec.sh (hunk 1 above)
Comment (xpivarc) [reply hack/gosec.sh]: done

**Review by enp0s3 — APPROVED** [review]
/lgtm

**Pull request merged**